examples/scripts/observable_job.py

```diff
@@ -32,7 +32,7 @@
 # Constructing the circuit by adding gates and measurements
 circuit.add(H(0))
 circuit.add(Rx(1.76, 1))
-circuit.add(ExpectationMeasure([0, 1], observable=obs, shots=1000))
+circuit.add(ExpectationMeasure(obs, [0, 1], shots=1000))
```

Comment:
if targets are optional, should leave them here ?

Reply:
same question for all the following examples where targets involved span the whole circuit

Reply:
done